kernel/object-store/src/main/java/io/mosip/commons/khazana/impl/S3Adapter.java

```diff
@@ -353,7 +351,7 @@ private AmazonS3 getConnection(String bucketName) {
             List<ObjectDto> objectDtos = new ArrayList<>();
             os.forEach(o -> {
                 // ignore the Tag file
-                String[] tempKeys = o.getKey().endsWith(TAGS_FILENAME) ? null : o.getKey().split("/");
+				String[] tempKeys = o.getKey().contains(TAGS_FILENAME) ? null : o.getKey().split("/");

```

Comment:
this one is risky, since if there is any file inbetween has tags words, then that will also be removed

Reply:
Fix : if (tempKeys[1] != null && tempKeys[1].endsWith(TAGS_FILENAME))
					tempKeys = null;